rever/activities/bibtex.xsh

```diff
@@ -0,0 +1,75 @@
+"""Activity for bumping version."""
+import datetime
+import os
+
+from rever import vcsutils
+from rever.activity import Activity
+try:
+    import bibtexparser
+except ImportError:
+    pass
```

Comment:
Generally, it is better to set `bibtexparser = None`, here so that it can be tested whether or not the import was successful later on.